docs/docsite/rst/tips_tricks/index.rst

```diff
@@ -0,0 +1,21 @@
+.. _tips_tricks_index:
+
```

Comment:
Please always keep the old anchors around even if you want to add a new one like this.  While it's an extreme corner case, these anchors also work from a collection so if we remove it and a collection WAS using it, that's a broken link now.  See https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-and-other-format-macros-within-module-documentation  - specifically the ``R()`` option.

Reply:
Thanks for the heads up @samccann